src/traces/contour/defaults.js

```diff
@@ -33,7 +33,7 @@ module.exports = function supplyDefaults(traceIn, traceOut, defaultColor, layout
 
     var contourStart = Lib.coerce2(traceIn, traceOut, attributes, 'contours.start'),
         contourEnd = Lib.coerce2(traceIn, traceOut, attributes, 'contours.end'),
-        autocontour = coerce('autocontour', !(contourStart && contourEnd));
+        autocontour = coerce('autocontour', contourStart === false || contourEnd === false);
```

Comment:
Great. @monfera would you mind adding a test case for this in the [`contour_test.js`](https://github.com/plotly/plotly.js/blob/85dacd78c02c35f3adc1d792c585e01855451966/test/jasmine/tests/contour_test.js#L28) suite?